src/test/java/petStoreTests/user/FindUserNegativeTest.java

```diff
@@ -0,0 +1,46 @@
+package petStoreTests.user;
+
+import client.UserClient;
+import data.DataSet;
+import dto.requests.user.User;
+import io.qameta.allure.Description;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+import petStoreTests.AbstractTest;
+
+import java.io.IOException;
+
+public class FindUserNegativeTest extends AbstractTest {
+
+    @Description("Verify that we are can find user in the store data base")
+    @Parameters({"User for adding and finding"})
+    @Test(dataProvider = "testData")
+    public void findUserTest(User user) throws IOException {
+
+        LOGGER.info("BEFORE TEST ADD USER");
+        UserClient.deleteUserByUsername(user.getUsername());
```

Comment:
Logger says you are adding a user, but I see a call of `deleteUserByUsername()` method